controlpanel/api/cluster.py

```diff
@@ -252,7 +274,24 @@ def install(self, **kwargs):
             escaped_val = val.replace(',', '\,')
             set_values.extend(['--set', f'{key}={escaped_val}'])
 
+        return set_values
+
+    def install(self, **kwargs):
+        # Uninstall release with old naming scheme
+        found = helm.list_releases(self.old_release_name)
+        for release in found:
+            # helm list if very fuzzy and could
+            # return other releases containing the release
+            # name we're looking for.
+            # To avoid accidentally uninstall a different
+            # release we check equality here.
+            if release == self.old_release_name:
+                helm.delete(True, self.old_release_name)
+                break
```

Comment:
I may actually move this little bit of logic into a `_delete_legacy_release()` helper function.
This way I can also get rid of that `old_release_name` property only used by this and use a simple variable.

Reply:
For the record, this should have been a "private" property called `_old_release_name` anyway instead to signal the fact it is not to be used outside this module. (but it doesn't matter as I'm going to get rid of it anyway)